src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -302,7 +319,18 @@ protected String getGLDebitCreditCode(String saeDebitCreditCode) {
             case ConcurConstants.DEBIT :
                 return KFSConstants.GL_DEBIT_CODE;
             default :
-                throw new IllegalArgumentException("Unrecognized debit/credit code: " + saeDebitCreditCode);
+                throw new IllegalArgumentException("Unrecognized SAE debit/credit code: " + saeDebitCreditCode);
+        }
+    }
+
+    protected String getOppositeGLDebitCreditCode(String glDebitCreditCode) {

```

Comment:
One of my updates from yesterday removed this method for now. Any work for adjusting the code based on signage will be handled by future stories.